packages/patternfly-3/react-console/src/DesktopViewer/ConnectWithRemoteViewer.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import MoreInformationContent from './MoreInformationContent';
+import { generateVVFile, downloadFile } from './vvFileGenerator';
+import consoleDetailPropType from './consoleDetailPropType';
+
+class MoreInformation extends React.Component {
```

Comment:
move this to another file? maybe it should even exists in pf?

Reply:
Done - moved to a separate file.
I'll open PR adding this component to pf-react and if accepted, I'll refactor here.